tasks/bench.py

```diff
@@ -80,6 +80,11 @@ def dogstastd(ctx):
 
     ctx.run("{} -pps=5000 -dur 45 -ser 5 -brk -inc 1000 {}".format(bin_path, options))
 
+# Temporarily keep compatibility after typo fix
+@task(pre=[build_dogstatsd])
+def dogstastd(ctx):
+    dogstatsd(ctx)
+
```

Comment:
😅

Reply:
What should we do? I don't know from where this task is called.